Lib/test/test_statistics.py

```diff
@@ -2312,6 +2312,72 @@ def test_error_cases(self):
             quantiles([10, None, 30], n=4)      # data is non-numeric
 
 
+class TestMultivariateStatistics(unittest.TestCase):
+    
+    def test_unequal_size(self):
+        for x, y in [
+            ([1, 2, 3], [1, 2]),
+            ([1, 2], [1, 2, 3]),
+        ]:
+            with self.assertRaises(self.module.StatisticsError):
+                statistics.covariance(x, y)
+            with self.assertRaises(self.module.StatisticsError):
+                statistics.covariance(x, y)
```

Comment:
Copy/paste error here, testing `covariance()` twice but not `correlation()`. Below as well.